apps/fz_http/lib/fz_http/conf/saml_config.ex

```diff
@@ -0,0 +1,21 @@
+defmodule FzHttp.Conf.SAMLConfig do
```

Comment:
These virtual schemas are so nice, thanks Ecto!

Reply:
The current validations are very simple. We can expand them in the future to validate more, for example, url pattern and xml validity.